PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/intellij/ui/ActivityLogToolWindowFactory.java

```diff
@@ -89,41 +85,6 @@ public void mouseClicked(MouseEvent e) {
     }
 
     public void registerDeploymentListener() {
-        AzurePlugin.addDeploymentEventListener(
```

Comment:
may need wait hdinsight validation to make sure this feature has been deprecated